README.md

```diff
@@ -1,18 +1,20 @@
 # `hardware.inc`
 ### Gameboy Hardware definitions
-`hardware.inc` has been the standard include file containing Game Boy hardware definitions for use in [rgbds](https://github.com/gbdev/rgbds) projects for over 20 years.
+`hardware.inc` has been the standard include file containing Game Boy hardware definitions for use in [rgbds](https://github.com/gbdev/rgbds) projects for over 20 years. Our intent here at gbdev is to continue to use `hardware.inc` as the standard as it has been for all these years.
```

Comment:
```suggestion
`hardware.inc` has been the standard include file containing Game Boy hardware definitions for use in [RGBDS](https://rgbds.gbdev.io) projects for over 20 years.
```
I think that this is self-obvious enough, and if it really needs to be said, it should be said after the historical lines below?